src/olympia/reviewers/utils.py

```diff
@@ -479,8 +479,6 @@ def set_addon(self, **kw):
 
```

Comment:
FIXME: also make sure in `confirm_auto_approved` we set `confirmed` on unlisted as well.

Reply:
c29507e566113d736ea301cba7efd87f53584840